README.md

```diff
@@ -8,25 +8,47 @@ JavaScript Closures Lab
 3. Use a closure to define a partially applied function
 
 ## Introduction
-
-- Include a brief review of the previous lesson (https://github.com/learn-co-curriculum/javascript-closures)
-- Be sure to spend some time talking about partially applied functions
-- These things can be hard/weird to test — we're sort of violating a lot of testing principles by testing implementation instead of behavior, which can go against an experienced programmer's intuition
-- It might be worth setting up contrived examples for the first few practice problems
-  - A simple curried example:
-
-  ```javascript
-  function adder(x) {
-    return function(y) {
-      return x + y
-    }
-  }
-
-  var add5 = adder(5)
-  add5(4) // 9
-  ```
-
-- Remember to push `master` and `solution` branches separately
+In the README, we saw what closures were and how we could leverage them to write better code. Time to put that knowledge
```

Comment:
Rather than referencing the README explicitly, could we say something general about closures? This lesson might get reused in other courses, and it may or may not come after that particular README -- it's like decoupling, but with lessons :)